packages/components/table/src/table-body/index.ts

```diff
@@ -64,7 +64,7 @@ export default defineComponent({
   render() {
     const { wrappedRowRender, store } = this
     const data = store.states.data.value || []
-    return h('tbody', {}, [
+    return h('tbody', { tabIndex: -1 }, [
```

Comment:
Thanks, would you mind adding some comments for this code?

Reply:
It is used to capture focus events triggered by the mouse

Reply:
Ok, I will add the link from MDN